### ISSUE TITLE
Consider marking measurements affected by software bugs with a flag

### ISSUE THREAD

**Issue by hellais** [ooni/pipeline#153]
There are some measurements that present installation software bugs and we should be ignoring those when processing data.

An example of this is installation issues with pyopenssl that present `http_experiment_failure` as `unknown_failure 'module' object has no attribute 'OP_SINGLE_ECDH_USE'` see: https://explorer.ooni.torproject.org/measurement/20190424T123447Z_AS24863_kIRqlMN1n478mA8aHwvioi6ofIWlasCcJtabGeh255a3CmdzY6?input=https:%2F%2Fexplorer.ooni.io.

As we find more of these it's perhaps useful to add them to this issue so we don't loose them.

**Comment by darkk** [comment]
Yep, all the 345 `https://` urls in the report show same error, so the error is unlikely triggered by the network.

> As we find more of these it's perhaps useful to add them to this issue so we don't loose them.

I assume that adding 294'093 measurements to the ticket is not the thing you actually want :-)

```
metadb=# select now()::date, count(*) from http_verdict where http_experiment_failure like '%OP_SINGLE_ECDH_USE%';
    now     | count  
------------+--------
 2019-04-25 | 294093
```
